app/views/smart_proxies/show.html.erb

```diff
@@ -7,6 +7,9 @@
     <% if service_features.any? %>
       <li><a href="#services" data-toggle="tab"><%= _('Services') %></a></li>
     <% end %>
+    <% sorted_pagelets_at(:tabbed_menu).each do |tab| %>
```

Comment:
Nitpick: This helper could be named like:

    <% extension_point_for(:tabbed_menu).each do |pagelet| %>

I also kinda don't like `tabbed_menu` name, how about:

    <% extension_point_for(:main_tabs).each do |pagelet| %>

I suggest to use plurals for all extension point names (as they are all collections).